src/Microsoft.Net.Http.Headers/project.json

```diff
@@ -24,6 +24,7 @@
   "frameworks": {
     "netstandard1.1": {
       "dependencies": {
+        "Microsoft.Extensions.Primitives": "1.1.0-*",
```

Comment:
Shouldn't `NETStandard.Library` be moved to be framework-specific? Like https://github.com/khellang/HttpAbstractions/commit/b5f8a8d5c7c3d3406c631ce64d521fa0a3de6f53?